core/tests/test_utils.py

```diff
@@ -442,7 +540,15 @@ def mock_delete_by_query(self, index_name, query):
             'deleted': index_size
         }
 
-    def mock_search(self, body=None, index=None, params=None):
+    # Here Any type is used because argument 'body' can accept dictionaries
+    # that can possess different types of values like int, List[...]. nested
+    # dictionary and other types too.
+    def mock_search(
```

Comment:
should we update the docstring? the argument definition doesn't show that its `Optional`

Reply:
Done !